src/detail/flare.cc

```diff
@@ -76,7 +100,7 @@ size_t flare::extinguish() {
   char tmp[stack_buffer_size];
   size_t result = 0;
   for (;;) {
-    auto n = ::read(fds_[0], tmp, stack_buffer_size);
+    auto n = PIPE_READ(fds_[0], tmp, stack_buffer_size);
     if (n > 0)
       result += static_cast<size_t>(n);
     else if (n == -1 && errno == EAGAIN)
```

Comment:
This usage of `EAGAIN` didn't change to use `try_again_later()` like other places, that just accidental?

Reply:
That's an oversight. Fixed.